tests/plaid/test_plaid.py

```diff
@@ -0,0 +1,231 @@
+        self.prm = PlaidParams().__dict__['prm']
+        self.prd = PlaidParams().__dict__['prd']
+        self.mtd = PlaidMetadata().__dict__['mtd']
+
+    def tearDown(self):
+        ''' reset test values after running tests'''
+
+        self.fb = None
+        self.prm = None
+        self.prd = None
+        self.mtd = None
+
+    def test_credit(self):
+        ''' perform test calling app functions '''
+
+        s, f = plaid_credit_metrics(self.fb, self.prm, self.mtd, self.prd)
+
+        self.assertCountEqual(s, self.expected)
+        self.assertListEqual(s, self.expected)
+
+
+class TestMetricVelocity(unittest.TestCase):
+
+    def setUp(self):
+        ''' import test values (inputs) that will feed app functions '''
+
+        # expected result
+        self.expected = [0, 0, 0, 1.0, 0.17]
+
+        # inputs
+        self.fb = PlaidFeedback().__dict__['fb']
+        self.prm = PlaidParams().__dict__['prm']
+        self.mtd = PlaidMetadata().__dict__['mtd']
+
+    def tearDown(self):
+        ''' reset test values after running tests'''
+
+        self.fb = None
+        self.prm = None
+        self.mtd = None
+
+    def test_velocity(self):
+        ''' perform test calling app functions '''
+
+        s, f = plaid_velocity_metrics(self.fb, self.prm, self.mtd)
+
+        self.assertCountEqual(s, self.expected)
+        self.assertListEqual(s, self.expected)
+
+
+class TestMetricStability(unittest.TestCase):
+
+    def setUp(self):
+        ''' import test values (inputs) that will feed app functions '''
+
+        # expected result
+        self.expected = [0.17, 0.31]
+
+        # inputs
+        self.fb = PlaidFeedback().__dict__['fb']
+        self.prm = PlaidParams().__dict__['prm']
+        self.mtd = PlaidMetadata().__dict__['mtd']
+
+    def tearDown(self):
+        ''' reset test values after running tests'''
+
+        self.fb = None
+        self.prm = None
+        self.mtd = None
+
+    def test_stability(self):
+        ''' perform test calling app functions '''
+
+        s, f = plaid_stability_metrics(self.fb, self.prm, self.mtd)
+
+        self.assertCountEqual(s, self.expected)
+        self.assertListEqual(s, self.expected)
+
+
+class TestMetricDiversity(unittest.TestCase):
+
+    def setUp(self):
+        ''' import test values (inputs) that will feed app functions '''
+
+        # expected result
+        self.expected = [0.9, 0.17]
+
+        # inputs
+        self.fb = PlaidFeedback().__dict__['fb']
+        self.prm = PlaidParams().__dict__['prm']
+        self.mtd = PlaidMetadata().__dict__['mtd']
+
+    def tearDown(self):
+        ''' reset test values after running tests'''
+
+        self.fb = None
+        self.prm = None
+        self.mtd = None
+
+    def test_diversity(self):
+        ''' perform test calling app functions '''
+
+        s, f = plaid_diversity_metrics(self.fb, self.prm, self.mtd)
+
+        self.assertCountEqual(s, self.expected)
+        self.assertListEqual(s, self.expected)
```

Comment:
Same here. At the moment our test suite comprises of 8 total assert statements (half of which -CountEqual- are implied by the other half -ListEqual-). It would be interesting to test for the functionality of the code generating the user's score, instead of just looking at the score output. This is nonetheless a good start though!